acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuNode.java

```diff
@@ -163,6 +168,20 @@ public void waitForNewFinalization() {
         MINUTES);
   }
 
+  public void waitForSyncCommitteeAggregate() {
+    LOG.debug("Wait for full sync committee aggregates");
+    waitFor(
+        () -> {
+          final Optional<SignedBlock> block = fetchHeadBlock();
+          if (block.isPresent()) {
+            final SignedBeaconBlockAltair altairBlock = (SignedBeaconBlockAltair) block.get();
+            assertThat(altairBlock.getMessage().getBody().syncAggregate.syncCommitteeBits).isEqualTo(Bytes.fromHexString("0xffffffff"));
+          }
+        },
```

Comment:
This will pass incorrectly if the block isn't present.  Probably need to just `assertThat(block).isPresent()`